package.json

```diff
@@ -75,7 +75,7 @@
       ],
       "category": "Utility",
       "synopsis": "Free and Open Source Password Vault",
-      "artifactName": "${productName}-${os}-x64.${ext}",
+      "artifactName": "${productName}-${os}-${env.NODE_ARCHITECTURE}-${version}.${ext}",
```

Comment:
Please remove the `version` parameter - AppImage builds can't have the version for updating correctly.